ci_framework/roles/install_yamls/defaults/main.yml

```diff
@@ -35,3 +35,22 @@ cifmw_install_yamls_whitelisted_vars:
   - OUTPUT_BASEDIR
   - OUTPUT_DIR
   - SSH_KEY
+  - OPENSTACK_OP_IMG
```

Comment:
I don't really get why those parameters have to be listed here. AFAIK they aren't taken by install_yamls, are they? I'd expect the list to be provided by some ansible env file that we load and consume?

Reply:
Yes, they are not used by install_yamls. From what I understand, the only image used for deployment is `OPENSTACK_IMG`.  I collected all the other images, just to be able to cross check the values in the new deployment. I can move them under a different env variable. Does that sound okay?

Reply:
I don't think using environment variables is needed/correct actually. Since the comparison is done in ansible, using actual ansible parameter should be sufficient ? You can create a new parameter as a dict and populate it, or even write a file in the artifacts directory (from the env_op_images for instance), and load it later (from the override_op_image). This would allow people to inject their own overrides, in addition.

What do you think?